src/components/questionary/questionaryComponents/NumberInput/QuestionTemplateRelationNumberInputForm.tsx

```diff
@@ -53,65 +57,93 @@ export const QuestionTemplateRelationNumberForm: FormComponent<QuestionTemplateR
         }),
       })}
     >
-      {formikProps => (
-        <>
-          <QuestionExcerpt question={props.field.question} />
-          <Field
-            name="config.small_label"
-            label="Small label"
-            type="text"
-            component={TextField}
-            margin="normal"
-            fullWidth
-            inputProps={{ 'data-cy': 'small-label' }}
-          />
-          <TitledContainer label="Constraints">
+      {formikProps => {
+        if (!getIn(formikProps.values, 'config.numberValueConstraint')) {
+          formikProps.setFieldValue(
+            'config.numberValueConstraint',
+            NumberValueConstraint.NONE
+          );
+        }
+
+        return (
+          <>
+            <QuestionExcerpt question={props.field.question} />
             <Field
-              name="config.required"
-              component={FormikUICustomCheckbox}
-              label="Check to make this field mandatory"
+              name="config.small_label"
+              label="Small label"
+              type="text"
+              component={TextField}
               margin="normal"
               fullWidth
-              InputProps={{ 'data-cy': 'required' }}
+              inputProps={{ 'data-cy': 'small-label' }}
```

Comment:
I see somewhere we use `InputProps` and here it starts with lowercase `inputProps`. Is it working in both cases and can we use the same form almost everywhere?

Reply:
There is some difference, according to the documentation:
```
inputProps | object |   | Attributes applied to the input element.
InputProps | object |   | Props applied to the Input element. It will be a FilledInput, OutlinedInput or Input component depending on the variant prop value.
```

Just we have some custom components so it's inconsistent as some properties are either not passed or missing the right types from the props.

Looking at the documentation we should use the lower case version as we want to set a custom attribute on the element, the upper case version can be used to set React props.